accesspoc/dips/views.py

```diff
@@ -427,26 +463,36 @@ def new_dip(request):
     if request.method == 'POST' and dip_form.is_valid() and dc_form.is_valid():
         dip = dip_form.save(commit=False)
         dip.dc = dc_form.save()
+        # TODO: Avoid this save from updating the related ES
+        # documents, as a later save when the async_result id
+        # is added, will repeat the process. It may require
+        # stop using signals and move to a custom save method.
```

Comment:
It'd be good to file a bug once it's merged! Unless you've already done it :)

Reply:
Created #79. I'll add more info after this is merged.